decidim-proposals/config/locales/en.yml

```diff
@@ -97,6 +99,8 @@ en:
             creation_enabled: Proposal creation enabled
             endorsements_blocked: Endorsements blocked
             endorsements_enabled: Endorsements enabled
+            hashtags_auto: Hashtags added to all proposals
```

Comment:
After reading the description, maybe `default_hashtags` would be a better naming?

Reply:
Mmm, I think that a default hashtag is something that is included at first, but that you can modify. In this case, these hashtags are fixed, they can't be removed from the proposal. This doesn't add too much value for the proposal inside the process (as all proposals have the same hashtags), but its a nice way to group proposals created in different participatory processes. For example, we are planning to use this to group proposals created in participatory processes at town level or at province level.
I can think two possible names for this:
* automatic_hashtags
* fixed_hashtags

I'd go with the first, but maybe you can think a better name

Reply:
`automatic_hashtags` sounds good!